example/logsignature_example.py

```diff
@@ -0,0 +1,102 @@
+######################
+# Here's how to code up a Neural CDE using the log-ode method for a long time series.
+# This method assumes familiarity with the standard Neural CDE example at `example.py`. We will only describe the
```

Comment:
`example.py` has been renamed.

Can you include a title + arxiv reference for the NRDE paper here?